### ISSUE TITLE
Don't make code wrap?

### ISSUE THREAD

**Issue by Hjdskes** [mtn/cocoa-eh-hugo-theme#14]
Hi!

Currently the theme wraps long lines in code. I prefer that it doesn't so I tried to change it. I added the following to my css overrides but it doesn't work. Do you know what is going wrong?

```css
section.main .content .markdown pre {
  overflow: auto;
  white-space: pre-wrap;
}

**Comment by fuegoio** [comment]
Hi !

This is what you should change : 

```css
@media (min-width: 600px) {
  .container {
    padding: 0 30px;
    width: 100%;
  }
}

section.main .content {
    width: 100%;
}

section.main .content .markdown pre {
    white-space: pre;
}
```
You need the media query too because the container has originally a `margin` attribute, but the `width: 100%` will cancel it so you need to add a `padding`.

**Comment by fuegoio** [comment]
Mhh I decided to look why you wanted that, and in fact I think that I prefer this version, on mobile it is just a nightmare to read the code. Really thanks you for your idea ^^

Fixed in 182d6ae7662d70c5c1375a0dd31b02e848ab8f8b

**Comment by Hjdskes** [comment]
Great!
